manifests/vhost/docroot.pp

```diff
@@ -0,0 +1,22 @@
+
+define apache::vhost::docroot (
+  Variant[Boolean,String] $docroot,
+  $virtual_docroot                                                                  = false,
+  $vhost                                                                            = $name,
+) {
+
+  if $virtual_docroot {
+    if ! defined(Class['apache::mod::vhost_alias']) {
```

Comment:
I know this was a pattern before, but ```include``` also works on a class that's already defined so I think this can be removed. I'd submit a PR for the existing code but that'd create huge conflicts for this PR for little benefit.